### PR TITLE
Feature/override for elevator and wrist

### PR DESCRIPTION
**Pull request by johnli01**

### CODE REVIEW

File: Competition2018/src/main/java/competition/subsystems/elevator/commands/ElevatorDangerousOverrideCommand.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    @Override
+    public void initialize() {
+        log.info("Initializing");
+    }
+
+    @Override
+    public void execute() {
+        elevator.insanelyDangerousSetPower(oi.driverGamepad.getRightStickY());
```
Comment (JohnGilb) [inline Competition2018/src/main/java/competition/subsystems/elevator/commands/ElevatorDangerousOverrideCommand.java]: `operatorGamepad`

File: Competition2018/src/main/java/competition/subsystems/wrist/commands/WristDangerousOverrideCommand.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    }
+    
+    @Override
+    public void initialize() {
+        log.info("Initializing");
+    }
+
+    @Override
+    public void execute() {
+        wrist.insanelyDangerousSetPower(oi.driverGamepad.getLeftVector().y);
```
Comment (JohnGilb) [inline Competition2018/src/main/java/competition/subsystems/wrist/commands/WristDangerousOverrideCommand.java]: `operatorGamepad`

**Review by JohnGilb — CHANGES_REQUESTED** [review]

**Comment by JohnGilb** [comment]
Also, when this is running, you should disable the soft limits.

File: Competition2018/src/main/java/competition/subsystems/elevator/commands/ElevatorDangerousOverrideCommand.java (hunk 1 above)
Comment (johnli01) [reply Competition2018/src/main/java/competition/subsystems/elevator/commands/ElevatorDangerousOverrideCommand.java]: Is there a certain button I should place it on? or should I just find room for it?

File: Competition2018/src/main/java/competition/subsystems/wrist/commands/WristDangerousOverrideCommand.java (hunk 2 above)
Comment (johnli01) [reply Competition2018/src/main/java/competition/subsystems/wrist/commands/WristDangerousOverrideCommand.java]: Is there a certain button I should place it on? or should I just find room for it?

**Comment by johnli01** [comment]
What exactly are soft limits?

**Pull request merged**